### PR TITLE
Simplify

### PR DESCRIPTION
**Pull request by di**
This PR ditches the `ClassifierNode`/`ClassifierTree` entirely (along with the code generation) since we don't actually need the tree hierarchy for PyPI.

Tests now run against the same `classifiers`/`deprecated_classifiers` objects that end up in the package.

Also, switches to CalVer.

### CODE REVIEW

File: trove_classifiers/__init__.py [hunk 1]
```diff
@@ -703,6 +703,6 @@
 }
```
Comment (ewdurbin) [inline trove_classifiers/__init__.py:703]: First three lines of this file should be removed now?

**Comment by ewdurbin** [comment]
This is rad! I'm a big fan of the simplification, but have a couple nits:

- Can the ordering of the items `trove_classifiers/__init__.py` be enforced via test? I don't think it is crucial... would just help keep things sane. I'd say this is priority 1/10
- The structure of `deprecated_classifiers` is pretty vague. Now that it's not machine generated a comment describing it would probably get good mileage. priority 5/10, worth changing but not a blocker by any means.

File: trove_classifiers/__init__.py (hunk 1 above)
Comment (di) [reply trove_classifiers/__init__.py:700]: Good catch 👍

**Comment by di** [comment]
> Can the ordering of the items `trove_classifiers/__init__.py` be enforced via test?

If you have any idea how to do this, I'm all ears. But since these are sets/dicts respectively, it'll be hard to do w/o doing some static analysis of the file.

Maybe there's a flake8 plugin or something that enforces order for elements in unordered data structures?

**Comment by pradyunsg** [comment]
I mean... we can add a pytest test that reads the file, and checks if the set's contents are indeed correctly. It's OK to be *slightly* fragile at times.

PS: I'm only *slightly* crazy. :)

**Comment by di** [comment]
Created a new issue to capture this: #7.

**Pull request merged**